examples/virtual_imaging/creating_a_segmented_detector.py

```diff
@@ -0,0 +1,43 @@
+"""
+=============================
+Creating a Segmented Detector
+=============================
+
+In this example we will show how to create virtual images for a segmented-like detector.
+
+This is helpful for basic mapping of orientation etc. in a diffraction pattern and can be useful
+for a first look at the data.
+"""
+
+import pyxem as pxm
+from pyxem.utils._azimuthal_utils import _get_control_points
+import numpy as np
+import hyperspy.api as hs
+
+
+dp = pxm.data.tilt_boundary_data()
+dp.calibrate.center = None  # Center the diffraction patterns
+dp.calibrate.scale = 0.1  # Scale the diffraction patterns in reciprocal space
+
+
+# Visualizing the virtual detector
+cp = _get_control_points(
+    1, npt_azim=8, radial_range=(1, 5), affine=dp.calibrate.affine
+)[:, :, ::-1]
+poly = hs.plot.markers.Polygons(verts=cp, edgecolor="w", facecolor="none")
+dp.plot()
+dp.add_marker(poly)
+pos = np.mean(cp, axis=1)
+texts = np.arange(len(pos)).astype(str)
+texts = hs.plot.markers.Texts(offsets=pos, texts=texts, color="w")
+dp.add_marker(texts)
```

Comment:
Would it make sense to put this in a method or in standalone utility function so that user can take the same argument as  `get_azimuthal_integral2d` and add it automatically to the signal? Something along the line of:

```python
kwargs = dict(npt=1, npt_azim=8, radial_range=(2, 5))
markers = get_azimuthal_integral2d_marker(signal, **kwargs)
s.add_marker(markers)
az = dp.get_azimuthal_integral2d(**kwargs)
```

I don't know if there are other functionalities in pyxem dealing with marker that would be worth considering when implementing this?

Reply:
@ericpre Yea that was my original thought and I will probably add that.

I also wanted to add some more examples getting people comfortable with adding markers in pyxem. I guess I can do both :) and just add to the example.